### PR TITLE
updated announce.styl

### PR DESCRIPTION
**Pull request by MiniViolin**

### CODE REVIEW

File: orion/sites/admin/src/announce/announce.styl [hunk 1]
```diff
@@ -1,13 +1,48 @@
 @import "../static.styl"
 
+
```
Comment (ahsu1230) [inline orion/sites/admin/src/announce/announce.styl]: extra line

File: orion/sites/admin/src/announce/announce.styl [hunk 2]
```diff
@@ -1,13 +1,48 @@
...
+  font-family: fontFamilyRoboto /*not working*/
 
   & h1
     margin: spacing4x 0
     line-height: lineHeight3
     font-size: fontSize3
+  & ul
+    padding: 0
+
+  .announce-lists
```
Comment (ahsu1230) [inline orion/sites/admin/src/announce/announce.styl]: If there is an indent here, please add a & before it.
```
#view-announce
    & .announce-lists
```

File: orion/sites/admin/src/announce/announce.styl [hunk 3]
```diff
@@ -1,13 +1,48 @@
...
+
+    & .li-med
+      width: spacing16x
+      display: inline-block
+
+    & .li-large
+      width: spacing64x
+      display: inline-block
+
+  .announcement-button
```
Comment (ahsu1230) [inline orion/sites/admin/src/announce/announce.styl]: same here (add &)

File: orion/sites/admin/src/announce/announce.styl [hunk 4]
```diff
@@ -1,13 +1,48 @@
...
+    display: inline-block
+    list-style: none
+    padding-left: 0
+    height: lineHeight6
+    line-height: lineHeight6
+    border-bottom: 1px solid colorGrayLight
+
+    & .li-small
+      width: spacing8x
+      display: inline-block
```
Comment (ahsu1230) [inline orion/sites/admin/src/announce/announce.styl]: there's a `display: inline-block` in here, .li-med, and .li-large.
Can we use a common className to give all these `.li-x` elements `display: inline-block` so it only appears once?

File: orion/sites/admin/src/announce/announce.styl (hunk 1 above)
Comment (MiniViolin) [reply orion/sites/admin/src/announce/announce.styl]: Oh whoops, I'll do that for the next PR, but for the one I just pushed I hadn't seen this comment yet.

File: orion/sites/admin/src/announce/announce.styl (hunk 2 above)
Comment (MiniViolin) [reply orion/sites/admin/src/announce/announce.styl]: I'm not sure I understand what you mean. Do you mean to indent the ul or and & and indent the .announce-lists?

File: orion/sites/admin/src/announce/announce.styl (hunk 4 above)
Comment (MiniViolin) [reply orion/sites/admin/src/announce/announce.styl]: Originally I just had the display:inline-block in the .announce-lists class, which is the "parent" class for .li-med, .li-large, and .li-small. However, this causes formatting issues in that the display: inline-block doesn't appear in the list elements.

**Pull request closed without merging**